plugins/clipboard/plugin.js

```diff
@@ -2539,6 +2580,227 @@
 			return undefined;
 		}
 	};
+
+	/**
+	 * Fallback dataTransfer object which is used together with {@link CKEDITOR.plugins.clipboard.dataTransfer}
+	 * for browsers supporting Clipboard API, but not supporting custom MIME types (Edge 16+ - #962).
+	 *
+	 * @since 4.8
+	 * @class CKEDITOR.plugins.clipboard.fallbackDataTransfer
+	 * @constructor Creates a class instance.
+	 * @param {Object} [nativeDataTransfer] A native data transfer object.
+	 * @param {String} [customDataFallbackType=text/html] A MIME type used for storing custom data types.
```

Comment:
Default string value should be wrapped with quotes.

Reply:
Actually is there even a need to parametrize this? We're not using a different value here in plugin code, right? You could make it a readonly private property to keep it as variable, but I think there's no need to expose changing it as there is no real use case.